code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -853,6 +902,7 @@
 /datum/tarot/reversed/the_stars
 	name = "XVII - The Stars?"
 	desc = "May your loss bring fortune."
+	extended_desc = "will deal a large ammount of genetic damage to the user, as well as giving a random limb either a fracture, internal bleeding, or a burn wound. However, such cost is rewarding, as the user will draw two additional cards."
```

Comment:
```suggestion
	extended_desc = "will cause a large amount of genetic decomposition to the user, as well as hurting a limb. However, it will reward the user with two additional cards."
```